index.html

```diff
@@ -2641,7 +2641,8 @@ <h1>
     <p>
 It is critically important to apply the principles of Privacy by Design
 to all aspects of decentralized identifier architecture, because DIDs
-and DID Documents are &mdash; by design &mdash; administered directly by their controllers.
+and DID Documents are &mdash; by design &mdash; administered directly by
+the <a>DID controller</a>.
```

Comment:
```suggestion
the <a>DID controller(s)</a>.
```

Reply:
Github is doing something weird. The `<span ... /span>` markup doesn't belong; only the embedded `(s)`